config/default/capm3/namespace.yaml

```diff
@@ -2,3 +2,11 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: capm3-system
+  labels:
+    kubernetes.io/metadata.name: capm3-system
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: v1.27
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: restricted
+    pod-security.kubernetes.io/warn-version: v1.27
```

Comment:
The last line lacks a newline character.

Reply:
@zhouhao3  fixed new line comment.

Reply:
@zhouhao3 added a new line